gemrb/core/Interface.cpp

```diff
@@ -631,7 +631,17 @@ Interface::~Interface() noexcept
 
 	// Removing all stuff from Cache, except bifs
 	if (!config.KeepCache) DelTree(config.CachePath, true);
-	
+
+	Cursors.clear();
```

Comment:
this is unnecessary.

Reply:
No because their dtors then run after https://github.com/gemrb/gemrb/blob/c77003fe0735869b00429e1118a09a307098d579/gemrb/core/Interface.cpp#L635

If you look at the Valgrind/gdb report in #1871 then it's apparently the reason of calling a freed SDL internals: https://github.com/gemrb/gemrb/blob/c77003fe0735869b00429e1118a09a307098d579/gemrb/plugins/SDLVideo/SDLSurfaceSprite2D.cpp#L295

Reply:
We can maybe also try to reorder the members of Interface but I'm not sure if there is a reason to explicitly shut down the video driver. DLL-stuff or something?

Reply:
`VideoDriver` isn't a member, so the answer would be to not do that there.

Reply:
Actually, I just remembered I added that because it was preventing GemRB from terminating on macOS. But that is probably something that can just get moved into the Mac specific code.